### PR TITLE
[#1317] create point coverage for single point shp

### PR DESCRIPTION
**Pull request by zhiyuli**
This pr contains supplementary changes to further fix issue  #1317 .
If a shp file contains only one point, feature res type will create a point-type coverage metadata instead of box-type.

### CODE REVIEW

**Comment by zhiyuli** [comment]
@pkdash can you please review this. The shp file (loganOutletmv.shp) that caused issue #1317 has been included in unit test as a testing sample.

**Comment by pkdash** [comment]
@zhiyuli will do.

File: hs_geographic_feature_resource/tests/test_geo_feature.py [hunk 1]
```diff
@@ -515,3 +516,13 @@ def test_metadata_element_pre_create_and_update(self):
                                                    element_name="geometryinformation",
                                                    request=request)
         self.assertTrue(data["is_valid"])
+
+    def test_single_point_shp(self):
+
+        shp_full_path = "hs_geographic_feature_resource/tests/single_point_shp/logan_Outletmv.shp"
+        meta_array, meta_dict = parsed_md_dict=parse_shp_zshp("", "", "", "", shp_full_path)
```
Comment (pkdash) [inline hs_geographic_feature_resource/tests/test_geo_feature.py]: Why we need 3 variables ```meta_array, meta_dict, parsed_md_dict```?. I think this statement should be:
```_, meta_dict = parse_shp_zshp("", "", "", "", shp_full_path)```

Comment (zhiyuli) [inline hs_geographic_feature_resource/tests/test_geo_feature.py]: @pkdash good catch. it was a typo. fixed

**Comment by pkdash** [comment]
+1

**Pull request merged**